clean_hard.sh

```diff
@@ -0,0 +1,20 @@
+printf "\033c"
+
+echo "1. cleaning root folder..."
+rm -fr node_modules
+rm -fr package-lock.json
+
+echo "2. cleaning lumin folder..."
+rm -fr lumin/.out
+rm -fr lumin/bin
+rm -fr lumin/node_modules
+rm -fr lumin/package-lock.json
+
+echo "3. cleaning reactnative folder..."
+rm -fr reactnative/ios/Pods
+rm -fr reactnative/ios/Podfile.lock
+rm -fr reactnative/ios/*.xcworkspace
+rm -fr reactnative/node_modules
+rm -fr reactnative/package-lock.json
+
+say -v Melina "clean done"
```

Comment:
what does this do?

Reply:
It cleans `npm install` results and when it's done says (via audio output device) `clean done` (with Melina lovely voice).
It's very useful as you can switch to different task and switch back when you hear this `voice notification`.

Reply:
I was specifically asking about the last line. The say command

Reply:
It just says 'clean done' on the mac machine